common/defs/support.json

```diff
@@ -2559,7 +2554,6 @@
       "eth:tPCTT": "(AUTO) exclude testnet",
       "eth:tPHOTON": "(AUTO) exclude testnet",
       "eth:tPLS": "(AUTO) exclude testnet",
-      "eth:tPOLIS": "(AUTO) exclude testnet",
```

Comment:
we should probably keep `tPOLIS` out either way, given that we exclude other testnets